server/models/db/worker.js

```diff
@@ -22,7 +22,16 @@ async function insertWorkerAccount(firstName, lastName, type, workerId, email, p
     return new Worker(workerData[0]);
 }
 
+//This method returns the first name, last name and other information about the worker 
+async function getWorkersByWorkerIds(workerIds) {
+    if (_.isNil(workerIds)) return null;
+    const queryStmt = `select * from service_worker where worker_id in (${workerIds});`;
```

Comment:
HI Simran,
To increase code consistency, I would recommend leveraging the tables constant file for the table name, e.g., `Tables.worker`.

Reply:
Hi Amy! 

I've resolved this comment by changing the name as described by you! 

Thanks a lot! 

Simran